cms/migrations/0007_alter_insightstag_id_alter_link_id_alter_partner_id_and_more.py

```diff
@@ -0,0 +1,143 @@
+# Generated by Django 4.2.1 on 2023-05-16 08:44
```

Comment:
How well can we test this migration? Do we need to download the production database to check against?

Reply:
Yes, that's what I did, though I imagine the data I have is slightly out of date

Reply:
Sure that's fine. good work.